### DISCUSSION TITLE
看寒流大佬的视频里面的copilot怎么那么智能，我的就跟智障一样?

### DISCUSSION THREAD

**Discussion by oufu99** [BYJRK/DotNet-Discussions#302] [C# 技术交流]
看大佬演示json和host的那两节视频,用的Rider+copilot, 怎么他一下就知道是要实例化一个类,一下又知道要补全,感觉好智能。
然后我在tb弄了个远程激活的那种copilot在Visualstudio里面用，怎么提示就跟弱智一样，是我用法错了吗,还是啥原因,大佬指点一哈?
还是说因为是不同的IDE支持问题

**Comment by tsctsc6** [comment]
我猜是因为寒流录视频的时候写了几遍代码，于是扣牌楼就学会了（doge

**Reply by BYJRK** [reply]
Copilot似乎没有这种现学的功能，不过它确实会从你当前的项目中寻找线索
